runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -1404,3 +1404,45 @@ TR_JITServerRelocationRuntime::copyDataToCodeCache(const void *startAddress, siz
    return coldCodeStart;
    }
 #endif /* defined(J9VM_OPT_JITSERVER) */
+
+void
+printAOTHeader(const void* aotHeaderAddress, char * buff, const size_t BUFF_SIZE)
+   {
+   memset(buff, 0, BUFF_SIZE*sizeof(char));
+   if (!aotHeaderAddress)
+      {
+      strncat(buff, "null", BUFF_SIZE - strlen(buff) - 1);
+      return;
+      }
+
+   PORT_ACCESS_FROM_PORT(TR::Compiler->portLib);
+   OMRPORT_ACCESS_FROM_OMRPORT(TR::Compiler->omrPortLib);
+   TR_AOTHeader * hdrInCache = (TR_AOTHeader *)aotHeaderAddress;
+   OMRProcessorDesc processorDescription = hdrInCache->processorDescription;
+
+   int rowLength = 0;
+   for (size_t i = 0; i < OMRPORT_SYSINFO_FEATURES_SIZE; i++)
+      {
+      for (int j = 0; j < 32; j++) 
```

Comment:
Can we use something symbolic for this `32` value? sizeof(*processorDescription.features)

Reply:
added `CHAR_BIT*size(processorDescription.features[i])`